dashboard/src/features/projects/workspaces/components/WorkspaceApps/WorkspaceApps.tsx

```diff
@@ -80,7 +80,7 @@ function AllWorkspaceApps() {
 
                 <Chip
                   size="small"
-                  label={project.plan.isFree ? 'Starter' : 'Pro'}
+                  label={project.plan.name}
```

Comment:
same here

Reply:
the local dashboard doesn't have access to this page

Reply:
👍